apps/wallet-mobile/src/features/ListTxHistory/useCases/ListTxHistory/TxHistory.tsx

```diff
@@ -40,48 +43,50 @@ export const TxHistory = () => {
   const handleOnRefresh = () => sync()
 
   return (
-    <LinearGradient
-      colors={isDark ? ['rgba(19, 57, 54, 1)', 'rgba(20, 24, 58, 1)', 'rgba(22, 25, 45, 1)'] : colors.gradient} // it fixes a weird bug
-      start={{x: isDark ? 0.5 : 0.5, y: isDark ? 0 : 0.5}}
-      end={{x: isDark ? 0 : 0, y: isDark ? 0.5 : 0}}
-      style={styles.root}
-    >
-      <Spacer height={headerHeight} />
-
-      <CollapsibleHeader expanded={expanded}>
-        <BalanceBanner />
-
-        <ActionsBanner disabled={isLoading} />
-      </CollapsibleHeader>
-
-      <View style={styles.panel}>
-        <Space height="lg" />
-
-        <Text style={styles.title}>{strings.title}</Text>
-
-        <Space height="xl" />
-
-        <LockedDeposit />
-
-        <Space height="md" />
-
-        {meta.implementation === 'cardano-bip44' && showWarning && (
-          <WarningBanner
-            title={strings.warningTitle.toUpperCase()}
-            icon={infoIcon}
-            message={strings.warningMessage}
-            showCloseIcon
-            onRequestClose={() => {
-              LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut)
-              setShowWarning(false)
-            }}
-            style={styles.warningNoteStyles}
-          />
-        )}
-
-        <TxHistoryList onScroll={onScroll} refreshing={isLoading} onRefresh={handleOnRefresh} />
-      </View>
-    </LinearGradient>
+    <SafeAreaView edges={['right', 'left']} style={styles.root}>
+      <LinearGradient
+        colors={isDark ? ['rgba(19, 57, 54, 1)', 'rgba(20, 24, 58, 1)', 'rgba(22, 25, 45, 1)'] : colors.gradient} // it fixes a weird bug
```

Comment:
@banklesss dark and light gradients should be present in the theme, can you check with product why the dark is not there ?

Reply:
this fixes a weird bug : (

Reply:
i picked manually the colors